xml/Microsoft.Build.Globbing/MSBuildGlobWithGaps.xml

```diff
@@ -146,8 +146,12 @@
         <Parameter Name="stringToMatch" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="stringToMatch">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="stringToMatch">The string to match. If the string represents a relative path, it will get normalized against the glob's root. Cannot be <see langword="null" />.</param>
+        <summary>Matches the given <paramref name="stringToMatch" /> against the glob.
+                Matching is path aware:
+                - slashes are normalized - arguments representing relative paths are normalized against the glob's root.
+                For example, the glob **/*.cs does not match ../a.cs, since ../a.cs points outside of the glob root.
+             Returns <see langword="false" /> if <paramref name="stringToMatch" /> contains invalid path or file characters&gt;</summary>
         <returns>To be added.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <summary>Matches the given <paramref name="stringToMatch" /> against the glob.</summary>
        <returns><see langword="false" /> if <paramref name="stringToMatch" /> contains invalid path or file characters; <see langword="true"/> otherwise.</returns>
        <remarks>Matching is path aware: Slashes are normalized - arguments representing relative paths are normalized against the glob's root. For example, the glob **/*.cs does not match ../a.cs, since ../a.cs points outside of the glob root.</remarks>
```